.github/workflows/nfr.yml

```diff
@@ -35,104 +38,142 @@ permissions:
   contents: read
 
 jobs:
+  vars:
+    name: Set up vars
+    runs-on: ubuntu-22.04
+    outputs:
+      test_label: ${{ github.event.inputs.test_label || 'all' }}
```

Comment:
Do we need the last part of these (`|| all`) if a default should already be set? Or could a user input an empty string?

Reply:
The last part in all of these is for when we run it on a schedule. It can't read the inputs there.